Sources/Application.swift

```diff
@@ -283,5 +288,24 @@ class Application {
             }
         }
     }
-
+    
+    /// Execute a local "slash" command.
+    ///
+    /// The currently supported list of local commands are;
+    ///
+    /// - /openurl {space separated list of numbers}: opens numbered link in default browser
+    ///
+    func executeLocalCommand(_ command: String) -> Bool {
+        if (command.hasPrefix("/openurl")) {
+            let pieces = command.components(separatedBy: [",", " "])
+            let linkNumbers = pieces.flatMap { Int($0) }
+            for linkNumber in linkNumbers {
+                let url = self.links[linkNumber - 1]
+                NSWorkspace.shared().open(NSURL(string: url)! as URL)
```

Comment:
Let's switch to Util.shell("open", "url"). I think AppKit/NSWorkspace.shared().open will not work on Linux.